cobaya/likelihoods/planck_2018_lowl/TT_native.py

```diff
@@ -0,0 +1,157 @@
+import os
+import numpy as np
+from cobaya.likelihoods.base_classes import InstallableLikelihood
+
+class TT_native(InstallableLikelihood):
+    """
+    Python translation of the Planck 2018 Gibbs TT likelihood (python Eirik Gjerløw, Feb 2023)
+    See https://wiki.cosmos.esa.int/planck-legacy-archive/index.php/CMB_spectrum_%26_Likelihood_Code
+
+    """
+
+    install_options = {"github_repository": "CobayaSampler/planck_native_data",
+                       "github_release": "v1",
+                       "asset": "planck_2018_lowT.zip",
+                       "directory": "planck_2018_lowT_native"}
+
+    type = "CMB"
+    aliases = ["lowT"]
+
+    _lmin = 2
+    _lmax = 29 # Could in principle be extended to 200
+    _delta_l = 1000
+    _nbins = 0
+
+
+    _data = {}
+    _data['cov'] = np.zeros((251, 251))
+    _data['cl2x'] = np.zeros((1000, 251, 3))
+    _data['mu'] = np.zeros(251)
+    _data['mu_sigma'] = np.zeros(251)
+
+    @classmethod
+    def get_bibtex(cls):
+        from cobaya.likelihoods.planck_2018_lowl.TT import TT
+        return TT.get_bibtex()
+
+    def get_requirements(self):
+        return {'Cl': {'tt': self._lmax}}
+
+    def initialize(self):
+        if self.get_install_options() and self.packages_path:
+            path = self.get_path(self.packages_path)
+            self._data['cov'][2:, 2:] = np.loadtxt(os.path.join(path, 'cov.txt'))
+            self._data['cl2x'][:, 2:, 0] = np.loadtxt(os.path.join(path, 'cl2x_1.txt')) 
+            self._data['cl2x'][:, 2:, 1] = np.loadtxt(os.path.join(path, 'cl2x_2.txt')) 
+            self._data['cl2x'][:, 2:, 2] = np.loadtxt(os.path.join(path, 'cl2x_3.txt')) 
+            self._data['mu'][2:] = np.loadtxt(os.path.join(path, 'mu.txt'))
+            self._data['mu_sigma'][2:] = np.loadtxt(os.path.join(path, 'mu_sigma.txt'))
+            _nbins = len(self._data['cl2x'][:, 0, 0])
+
+            # Bandlimit covariance matrix
+            for l in range(self._lmin, self._lmax+1):
+                for k in range(self._lmin, self._lmax+1):
+                    if abs(l-k) > self._delta_l:
+                        self._data[cov][l, k] = 0
```

Comment:
cov is undefined

Reply:
If it works, code is probably redundant?

Reply:
I'll remove this part! Indeed it is redundant.